python/beeai_framework/context.py

```diff
@@ -134,7 +137,17 @@ def enter(
         context = RunContext(instance=instance, context_input=context_input, parent=parent)
 
         async def handler() -> R:
-            emitter = context.emitter.child(namespace=["run"], creator=context, context={"internal": True})
+            emitter = context.emitter.child(
+                namespace=["run"],
+                creator=context,
+                context={"internal": True},
+                events={
+                    "start": NoneType,
```

Comment:
The `start` event contains data or not?

Reply:
Not this one, this is internal to the RunContext itself and not public-facing. This is what the TS version does as well